nix/status-go/default.nix

```diff
@@ -51,21 +53,52 @@ let
       };
 
   mobileConfigs = {
-    android = {
-      name = "android";
-      outputFileName = "status-go-${srcData.shortRev}.aar";
-      envVars = [
-        "ANDROID_HOME=${androidPkgs.androidsdk}/libexec/android-sdk"
-        "ANDROID_NDK_HOME=${androidPkgs.ndk-bundle}/libexec/android-sdk/ndk-bundle"
-        "PATH=${makeBinPath [ openjdk ]}:$PATH"
-      ];
-      gomobileExtraFlags = [ "-androidapi 18" ];
-    };
-    ios = {
+    android =
+      let
+        androidLevel = if enableNimbus then "23" else "18"; # Target Android API level 23 when linking with Nimbus to avoid undefined stderr/stdout symbols when linking
```

Comment:
Wouldn't it make more sense to just bump everything to 23? 6.0(23) is more than 4 years old.

Reply:
Besides, we already use `23` as a minimum in Gradle:
https://github.com/status-im/status-react/blob/f34a7bc3f132b61b5c5cbd27d4aa773b392885f4/android/gradle.properties#L24